Makefile

```diff
@@ -451,18 +456,37 @@ ifeq ($(USE_FOLLY),1)
 	ifeq (,$(FOLLY_DIR))
```

Comment:
Now that we have FOLLY_PATH, FOLLY_DIR is confusing. Perhaps call it FOLLY_SRC. It would be nice if FOLLY_SRC were optional (if libfolly.a and headers already available in FOLLY_PATH).